src/app/page.tsx

```diff
@@ -7,10 +12,11 @@ export const metadata: Metadata = {
 
 const Hub: NextPage = () => {
   return (
-    <div className="m-auto max-w-[1200px] py-36">
-      <h1 className="max-w-[900px] font-satoshi text-7xl font-black text-secondary-500">
-        Discover and Empower with Monitoring Solutions.
-      </h1>
+    <div className=" text-secondary-500">
```

Comment:
`text-secondary-500` you may want to add it into `body {}` in `globals.css` file